tests/playwright-test/esm.spec.ts

```diff
@@ -148,7 +148,6 @@ test('should use source maps', async ({ runInlineTest, nodeVersion }) => {
 });
 
 test('should show the codeframe in errors', async ({ runInlineTest, nodeVersion }) => {
-  test.fixme();
```

Comment:
😎 Nice! I know I added this test specifically for the ESM case, but do we have one testing the non-ESM case, too?

Reply:
Yes, we do have: tests/playwright-test/reporter-base.spec.ts:66